src/odemis/driver/actuator.py

```diff
@@ -1307,13 +1309,14 @@ def __init__(self, name, role, dependencies, axis_name, positions, cycle=None,
 
         self._cycle = cycle
         self._move_sum = 0
-        self._position = {}
-        self._referenced = {}
+        self._position: Dict[str, float] = {}
```

Comment:
This variable is basically never used. We could entirely get rid of it by merging `updatePosition()` into  `_update_dep_position()`. This would especially avoid having the confusing `_positions` vs `_position`.

Could you clean it up?